packages/grid_client/scripts/applications/peertube.ts

```diff
@@ -0,0 +1,115 @@
+  const resultVM = await client.machines.deploy(vms);
+  log("================= Deploying VM =================");
+  log(resultVM);
+  log("================= Deploying VM =================");
+
+  const vmPlanetary = (await client.machines.getObj(vms.name))[0].planetary;
+  //Name Gateway Model
+  const gw: GatewayNameModel = {
+    name: subdomain,
+    node_id: gatewayNode.nodeId,
+    tls_passthrough: false,
+    backends: ["http://[" + vmPlanetary + "]:9000"],
+  };
+
+  const resultGateway = await client.gateway.deploy_name(gw);
+  log("================= Deploying name gateway =================");
+  log(resultGateway);
+  log("================= Deploying name gateway =================");
+}
+
+async function getDeployment(client, vms, gw) {
+  const resultVM = await client.machines.getObj(vms.name);
+  const resultGateway = await client.gateway.getObj(gw);
+  log("================= Getting deployment information =================");
+  log(resultVM);
+  log(resultGateway);
+  log("https://" + resultGateway[0].domain);
+  log("================= Getting deployment information =================");
+}
+
+async function cancel(client, vms, gw) {
+  const resvm = await client.machines.delete(vms);
+  const resgw = await client.gateway.delete_name(gw);
+  log("================= Canceling the deployment =================");
+  log(resvm);
+  log(resgw);
+  log("================= Canceling the deployment =================");
+}
+
+async function main() {
+  const name = "newpeertube";
+  const grid3 = await getClient(`peertube/${name}`);
+  const subdomain = "pt" + grid3.twinId + name;
+
+  //VMNode Selection
+  const vmQueryOptions: FilterOptions = {
+    cru: 1,
+    mru: 2,
+    sru: 50,
+    availableFor: grid3.twinId,
+    farmId: 1,
+  };
+  //GatewayNode Selection
+  const gatewayQueryOptions: FilterOptions = {
+    gateway: true,
+    availableFor: grid3.twinId,
+  };
+  const gatewayNode = (await grid3.capacity.filterNodes(gatewayQueryOptions))[0];
+
+  const vms: MachinesModel = {
+    name,
+    network: {
+      name: "wedtest",
+      ip_range: "10.249.0.0/16",
+    },
+    machines: [
+      {
+        name: "peertube",
+        node_id: +(await pingNodes(grid3, await grid3.capacity.filterNodes(vmQueryOptions))),
+        disks: [
+          {
+            name: "wedDisk",
+            size: 50,
+            mountpoint: "/data",
+          },
+        ],
+        planetary: true,
+        public_ip: false,
+        public_ip6: false,
+        mycelium: false,
+        cpu: 1,
+        memory: 1024 * 2,
+        rootfs_size: 0,
+        flist: "https://hub.grid.tf/tf-official-apps/peertube-v3.1.1.flist",
+        entrypoint: "/sbin/zinit init",
+        env: {
+          SSH_KEY: config.ssh_key,
+          PEERTUBE_WEBSERVER_HOSTNAME: subdomain + "." + gatewayNode.publicConfig.domain,
+          PEERTUBE_ADMIN_EMAIL: "admin123@peer.tube", // edit me
+          PT_INITIAL_ROOT_PASSWORD: "admin123", // edit me
```

Comment:
The `edit me` isn't a good way to tell the user that you can edit this section if you want tbh, instead, let's give the user a clear comment.

```ts
          // edit me
```